studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopover/NotificationRows.utils.tsx

```diff
@@ -88,14 +89,14 @@ export const formatNotificationText = (
       )
     } else if (upgrade_type === 'schema-migration') {
       const { version_to } = additional
-      if (ownerReassignStatus?.current === 'unmigrated') {
+      if (ownerReassignStatus?.desired === 'unmigrated') {
```

Comment:
@sweatybridge just to confirm so its the `desired` column that will be updated when the migration is applied/rolled back?

Reply:
Yes, desired state is always updated by api on apply/rollback. For active healthy projects, the current state will also be updated together with desired. For paused projects, api will only change desired state. The worker will move the project from `current` to `desired` when restoring.

So I think on FE, we need to check the desired state to indicate to users whether they have applied the migration to a paused project.